src/core/n-data.c

```diff
@@ -507,12 +507,15 @@ REBNATIVE(get)
     REFINE(2, opt);
 
     REBVAL *source = ARG(source);
+    REBVAL out;
+    VAL_INIT_WRITABLE_DEBUG(&out);
 
     if (ANY_WORD(source)) {
         *D_OUT = *GET_OPT_VAR_MAY_FAIL(source);
     }
     else if (ANY_PATH(source)) {
-        if (Do_Path_Throws(D_OUT, NULL, source, NULL))
+        Make_Block_Type(&out, REB_GET_PATH, FALSE, source);
+        if (Do_Path_Throws(D_OUT, NULL, &out, NULL))
```

Comment:
On a micro-optimization note (but Rebol is about trying to be optimized, in the long run...) the ARGs of a function (or native) are owned by that function.  They are copies of the REBVAL (4x size of pointer item) that the caller passed in, always.  So with things like this, if you know you are not going to use `source` again, you can just set the type bits in `source` without making a new variable.

(But note you have to VAL_SET_TYPE_BITS, which sets just the type...and not reset the entire header of the value!)